site/app/templates/grading/simple/Display.twig

```diff
@@ -246,9 +246,6 @@
 {% macro render_lab_cols(gradeable, section_id, grade, index) %}
     {% for component in grade.getGradeable().getComponents() %}
         {% if component.isText() %}
```

Comment:
If you delete the body of this if-statement, it would be cleaner to get rid of it entirely and replace the else-statement with its negation.